ref/wotsx1.h

```diff
@@ -0,0 +1,36 @@
+#if !defined( WOTSX1_H_ )
+#define WOTSX1_H_ 
+
+#include <string.h>
+
+/*
+ * This is here to provide an interface to the internal wots_gen_leafx1
+ * routine.  While this routine is not referenced in the package outside of
+ * wots.c, it is called from the stand-alone benchmark code to characterize
+ * the performance
+ */
+struct leaf_info_x1 {
+    unsigned char *wots_sig;
+    uint32_t wots_sign_leaf; /* The index of the WOTS we're using to sign */
```

Comment:
There is a 4 byte pad here.  Not sure if it makes any difference.  To get rid of it, move wots_sign_leaf to the end.

Reply:
Padding doesn't matter here (and is compiler/CPU specific in any case)